src/mscorlib/model.xml

```diff
@@ -8152,6 +8152,10 @@
       <Member MemberType="Property" Name="Local" />
       <Member MemberType="Property" Name="Utc" />
     </Type>
+    <Type Name="System.TimeZoneInfo+AdjustmentRule">
+      <!-- CreateAdjustmentRule doesn't have a public contract, but tests and other callers can use Reflection to invoke it, so ensure it is left in mscorlib. -->
```

Comment:
`Status="ApiFxInternal"` is meant to be used for these cases. It includes the implementation - so that it is reachable via reflection, but does not make it public.